compose/config/config.py

```diff
@@ -984,8 +994,11 @@ def has_uppercase(name):
 
 def load_yaml(filename):
     try:
-        with open(filename, 'r') as fh:
-            return yaml.safe_load(fh)
+        if is_compose_url(filename):
+            return yaml.safe_load(urllib.urlopen(filename))
```

Comment:
This is going to need a lot more error handling. There are a lot of failures that can happen here.